app/services/user_invite_service.rb

```diff
@@ -9,7 +9,6 @@ def self.invite(email:, resource:, name: nil, roles: [])
 
     user = User.find_by(email: email)
     if user
-      user.invite!
```

Comment:
How does this interact with the ability of the banks to resend the invitations to the partner?  Maybe invite! if the user has not yet accepted their invitation/set a password?

Reply:
There should be a way to force this behavior. Let me take a look.